docs/PORTING.md

```diff
@@ -38,6 +38,7 @@ To add a new `IOPlatform`:
 - `Window_Create`
 - `Window_PollEvents`
 - `Window_Swap`
+- `IOPlatform_Shutdown`
 - `Input_GetKeyMap`
 - `Input_IsKeyDown`
```

Comment:
Should all these functions have a `IOPlatform_` prefix? Right now it's a mishmash. (something to think about)